serverless/handler.py

```diff
@@ -81,6 +82,7 @@ def render(event, context):
                                headers=headers,
                                cookies=cookies,
                                html_encoding=html_encoding)
+        renderer.shutdown()
```

Comment:
```suggestion
        renderer.shutdown()
        del renderer
```
Just an idea. I know Python uses reference counting for garbage collection, but to ensure it is released we can be explicit. Might help, might not.